tests/CollectionTest.php

```diff
@@ -2062,6 +2066,77 @@ public function testIsVersioningEnabled()
         ));
         $this->assertFalse($this->database->col3->isVersioningEnabled());
     }
+
+    /**
+     * @throws Exception
+     */
+    public function testRenameNonExistentCollection()
+    {
+        $this->expectException(Exception::class);
```

Comment:
check travis builds. this test has failed on some environments. I think you need to use annotation instead of method

Reply:
Yes, I've seen those build errors with 5.4, 5.5 versions of php - although these methods are recommended for testing with PHPUnit lately, we need another solution to support backward compatibility.

Reply:
May you show me this recomendation? Try annotations

Reply:
https://stackoverflow.com/questions/5683592/phpunit-assert-that-an-exception-was-thrown?answertab=active#tab-top

Someone wrote even this:
+1 for not relying on the magic doc-block 

As for me, the best solution is when your lib/app works correctly on different platforms and for any user-space environment, so I'll make it with annotations.